.github/CODEOWNERS

```
@@ -71,6 +71,7 @@
 /pkg/api/ @grafana/grafana-backend-group
 /pkg/apis/ @grafana/grafana-app-platform-squad
 /pkg/apis/alerting_notifications @grafana/grafana-app-platform-squad @grafana/alerting-backend @grafana/alerting-frontend
+/pkg/apis/provisioning @grafana/grafana-app-platform-squad @grafana/grafana-git-ui-sync-team
```

```suggestion
/pkg/apis/provisioning @grafana/grafana-git-ui-sync-team
```
for now... and then we can figure it out